guides/common/modules/con_load-balancing-solution-architecture.adoc

```diff
@@ -28,7 +28,7 @@ In a load-balanced setup, nearly all {SmartProxy} functionality continues to wor
 Load balancer works with the following services and features:
```

Comment:
I recommend changing "Load balancer" to "A load balancer" or "Load balancing". A noun without an article looks strange at the beginning of a sentence.

Reply:
Fixed.